arches/app/media/js/viewmodels/card-component.js

```diff
@@ -125,7 +125,29 @@ define([
         };
 
         this.beforeMove = function(e) {
-            e.cancelDrop = (e.sourceParent!==e.targetParent);
+            if(e.sourceParent === e.targetParent) {
```

Comment:
We have the same problem in the graph tab, so maybe we should keep the underlying issue open until we tackle that, or did you want to roll in a fix?

Reply:
I'll look to see if it makes sense to do that here

Reply:
Yeah, I took care of the other situations as well related to node sort and card sort.